package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "express-generator",
   "description": "Express' application generator",
-  "version": "4.13.1",
+  "version": "4.14.0",
```

Comment:
It is common etiquette that a pull request never bumps a version, ever. Not sure why that would need to be spelled out. The pull request is only to add features, etc., not to dictate release schedules. Bumping the version here is missing a lot of things, for example, the version of this module follows the version on Express. I don't see this pull request bumping the Express dep to match the new version, and there are many other things that need to be done in this module before just bumping the version.

Are you not able to remove the version bump from the pull request?